publish_trips.py

```diff
@@ -97,13 +101,13 @@ def get_data(path, token):
     return res.text
 
 
-def handle_value(key, value, date_keys=["checkout_date"]):
+def handle_value(key, value, date_keys=["CheckoutDateLocal"]):
     if value == "":
         return None
     if not value or key not in date_keys:
         return value
     # Format a socrata-friendly date
-    return arrow.get(value, "M/D/YY").format("YYYY-MM-DD")
+    return dateutil.parser.parse(value).strftime("%Y-%m-%d")
```

Comment:
This actually wasn't doing anything because it was looking for the wrong key. The old keys are passed into here so we need to look for `CheckoutDateLocal`. Fixing this uncovered that some old files used "M/D/YYYY" instead so I dropped arrow for dateutil. dateutil's parser is a lot more flexible and can differentiate these two date formats.

Reply:
awesome that you caught this, and TIL `dateutil` 🚀